.github/workflows/vale-linter.yml

```diff
@@ -0,0 +1,19 @@
+name: Vale Linting
+
+on: pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  prose:
+    if: github.repository == 'keptn-sandbox/new-keptn-docs-engine'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```

Comment:
@iamrajiv any reason to use sha1 hash of commit instead of release tag for `checkout`?